airbyte-webapp/src/pages/connections/ConnectionStatusPage/ConnectionStatusPage.tsx

```diff
@@ -162,35 +162,31 @@ export const ConnectionStatusPage: React.FC = () => {
         title={
           <div className={styles.title}>
             <FormattedMessage id="sources.syncHistory" />
-            {connection.status === ConnectionStatus.active && (
+            {connection.status === ConnectionStatus.active && !activeJob?.action && (
               <div className={styles.actions}>
-                {!activeJob?.action && (
-                  <>
-                    <Button className={styles.resetButton} variant="secondary" onClick={onResetDataButtonClick}>
-                      <FormattedMessage id="connection.resetData" />
-                    </Button>
-                    <Button
-                      className={styles.syncButton}
-                      disabled={!allowSync}
-                      onClick={onSyncNowButtonClick}
-                      icon={
-                        <div className={styles.iconRotate}>
-                          <RotateIcon height={styles.syncIconHeight} width={styles.syncIconHeight} />
-                        </div>
-                      }
-                    >
-                      <FormattedMessage id="connection.startSync" />
-                    </Button>
-                  </>
-                )}
-                {activeJob?.action && !activeJob.isCanceling && cancelJobBtn}
-                {activeJob?.action && activeJob.isCanceling && (
-                  <Tooltip control={cancelJobBtn} cursor="not-allowed">
-                    <FormattedMessage id="connection.canceling" />
-                  </Tooltip>
-                )}
+                <Button className={styles.resetButton} variant="secondary" onClick={onResetDataButtonClick}>
```

Comment:
With my suggestion on the CSS there is no need for styles.resetButton nor styles.syncButton